core/services/ocr/config.go

```diff
@@ -4,13 +4,15 @@ import (
 	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting/types"
 
 	evmconfig "github.com/smartcontractkit/chainlink/v2/core/chains/evm/config"
+	"github.com/smartcontractkit/chainlink/v2/core/config"
 	"github.com/smartcontractkit/chainlink/v2/core/services/job"
-	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
 )
 
 // Config contains OCR configurations for a job.
 type Config interface {
-	pg.QConfig
+	Insecure() config.Insecure
```

Comment:
@dimriou Hmmmm... Not sure I fully understand why this was necessary? Are we reusing this interface somehow (and removing pg.QConfig?)

Reply:
We didn't use it because we mistakenly used AppConfig through ChainScoped instead, but now that we're switching to splitting these two we need this config.

Reply:
Currently it's being used in `chainlink/core/services/ocr/delegate.go`